src/app/preferences/general-preferences/general-preferences.module.ts

```diff
@@ -0,0 +1,12 @@
+/// <reference path="general-preferences.controller.ts"/>
+/// <reference path="general-preferences.service.ts"/>
+
+namespace Core {
+
+    export const generalPreferencesModule = angular
+      .module('hawtio-general-preferences', [])
+      .controller('GeneralPreferencesController', GeneralPreferencesController)
+      .service('GeneralPreferencesService', generalPreferencesService)
```

Comment:
Change the reference name of the service to uncapitalised. And revert the class name to Capitalised.

Reply:
I think this is resolved.